.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
This should probably be a copy of .gitignore shouldn't it?
Not that it's important.